core/error.go

```diff
@@ -38,17 +37,12 @@ type Error struct {
 // Error() allows *core.Error to present the `error` interface.
 func (e *Error) Error() string {
 	var ret string
+	ret = e.desc + "\n"
 
-	if os.Getenv("CONTIV_TRACE") != "" {
-		ret = e.desc + "\n"
-
-		for _, stack := range e.stack {
-			ret += fmt.Sprintf("%s [%s %d]\n", stack.fun, stack.file, stack.line)
-		}
-	} else {
-		ret = fmt.Sprintf("%s [%s %s %d]", e.desc, e.stack[0].fun, e.stack[0].file, e.stack[0].line)
+	// TODO: use github.com/pkg/errors
+	for _, stack := range e.stack {
+		ret += fmt.Sprintf("%s [%s %d]\n", stack.fun, stack.file, stack.line)
```

Comment:
https://stackoverflow.com/a/1766304/1322463

Reply:
this is what was in the code, this PR is just trying to remove the env var part, in fact this file shouldn't even exist, need to be refactor in other PR

Reply:
i see now it was relocated and not added, concur on refactor 👍